plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/Citadel.java

```diff
@@ -109,55 +109,61 @@ public void reload() {
 
     @Override
     public void onEnable() {
-        super.onEnable();
-        instance = this;
-        logger = getLogger();
-        if (!Bukkit.getPluginManager().isPluginEnabled("NameLayer")) {
-            logger.info("Citadel is shutting down because it could not find NameLayer");
-            Bukkit.shutdown();
-            return;
-        }
-        config = new CitadelConfigManager(this);
-        if (!config.parse()) {
-            logger.severe("Errors in config file, shutting down");
-            Bukkit.shutdown();
-            return;
-        }
-        typeManager = new ReinforcementTypeManager();
-        config.getReinforcementTypes().forEach(t ->
-        {
-            if (!typeManager.register(t)) {
-                logger.severe("Errors in the config file, shutting down");
+        try {
```

Comment:
Surrounding the code in a try catch is not necessary: if something within `onEnable()` throws, then it wont reach the `ready()` call

Reply:
I'd like to test that as I'm hesitant to believe it given recent events.

Reply:
It's part of the language. It's also evident in the stacktrace I included [here](https://github.com/CivMC/Civ/issues/777#issuecomment-2688345923) with the NPE with `this.activityMap`. It's null because it's set *after* config parsing, which is throwing.

Reply:
I didn't that deep into the stack trace, I'll remove it then